specs/image-update-monitoring.md

```diff
@@ -0,0 +1,32 @@
+# Docker Image Update Monitoring
+
+## Summary
+
+Many GitOps users would like to automate Kubernetes manifest changes in the deployment repository
+(see [Deployment Repo Update Automation](./deployment-repo-update.md)). The changes might be triggered by
+the CI pipeline run or a new image in the Docker registry. The [Flux] provides the docker registry monitoring as part of
+[Automated Image Update](https://docs.fluxcd.io/en/latest/references/automated-image-update.html) feature but that functionality
+is not available for non-Flux users and cannot be leveraged in manifests are generated by a script (e.g. jsonnet).
```

Comment:
> cannot be leveraged in manifests are generated by a script

In Flux it can. See https://docs.fluxcd.io/en/stable/references/fluxyaml-config-files.html#using-patch-updated-configuration

Reply:
```suggestion
is not available for non-Flux users and cannot be leveraged in manifests generated by a script (e.g. jsonnet).
```

Reply:
thank you! corrected the sentence.